int-test/tests/login.py

```diff
@@ -0,0 +1,62 @@
+from appium import webdriver
```

Comment:
I wonder if we want to use Node for test writing?
As the project itself is JavaScript based and is not guaranteed that contributors even know Python or have it installed.